docgen/content-sources/toc.yaml

```diff
@@ -1,148 +1,148 @@
 toc:
   - title: 'functions'
-    path: /docs/reference/functions/cloud_functions_.html
+    path: /docs/reference/functions/v1_cloud_functions_.html
```

Comment:
These paths make me realize that we have a lot of issues here. Every namespace's "overview" page lists paths that are no longer importable. Will talk more offline.

Reply:
I've done a crude fix to this that changes the header to a simpler "last segment" instead. I think it functions better than the status quo.